src/application/mat_list/action_generate_xls.php

```diff
@@ -18,6 +18,10 @@
  * along with eCamp.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+	# increase memory limit for generating xls
+    # keep overall memor_limit low to allow for more FPM processes per server (high WEB_CONCURRENCY)
+	ini_set("memory_limit","32M");

```

Comment:
Hier gibt es Beispiele, bei welchen 32M nicht aureicht.

Reply:
Konnte ich nicht reproduzieren. Habe be mir lokal auf 4M runtergeschraubt und konnte alle XLS exportieren. PDF ist mir schon viel früher abgeraucht. Ich stell mal beide auf 64M und dann schauen wir mal, wie es tut. Ok?